Marlin/Configuration_adv.h

```diff
@@ -449,7 +449,7 @@
 //#define LCD_DECIMAL_SMALL_XY
 
 // The timeout (in ms) to return to the status screen from sub-menus
-//#define LCD_TIMEOUT_TO_STATUS 15000
+#define LCD_TIMEOUT_TO_STATUS 60000
 
 #if ENABLED(SDSUPPORT)
 
```

Comment:
Don't include changes to the main configurations unless they pertain to the feature.